server/images.go

```diff
@@ -1162,3 +1165,22 @@ func verifyBlob(digest string) error {
 
 	return nil
 }
+
+var deployment = sync.OnceValue(func() string {
```

Comment:
This should most likely return a User-Agent such as `Docker` or `Docker/<version>` if we have that info.

Reply:
From within the container I don't think we have a good way to detect that, as that's basically a container escape vector.  We might be able to tease out "docker" vs. other runtimes, but I'd need to spin up a bunch of permutations to make sure I got all the parsing right, so I'd suggest we defer that for now.